src/Codata/Thunk.agda

```diff
@@ -8,12 +8,27 @@
 
 module Codata.Thunk where
 
+open import Level
 open import Size
-open import Relation.Unary
+
+private
+  variable
+    ℓ ℓ₁ ℓ₂ : Level
 
 ------------------------------------------------------------------------
 -- Basic types.
 
+SizedType : (ℓ : Level) → Set (suc ℓ)
+SizedType ℓ = Size → Set ℓ
```

Comment:
Shouldn't these additions go in `Size` rather than `Codata.Thunk`?

Reply:
Makes sense, thanks.